xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/WinVideoFilter.cpp

```diff
@@ -502,12 +511,16 @@ void COutputShader::CreateDitherView()
 
 //==================================================================================
 
-bool CYUV2RGBShader::Create(EBufferFormat fmt, COutputShader *pCLUT)
+bool CYUV2RGBShader::Create(EBufferFormat fmt, COutputShader *pCLUT, ESCALINGMETHOD scalingMethod)
 {
   m_format = fmt;
   m_pOutShader = pCLUT;
+  m_isNearest = scalingMethod == VS_SCALINGMETHOD_NEAREST;
```

Comment:
instead of storing this as a binary value, I think `m_scalingMethod` makes more sense. That way, future code could depend on this logic instead of having to add a binary variable for each shader type.